Assets/VRTK/SDK/HyperealVR/SDK_HyperealVRDefines.cs

```diff
@@ -0,0 +1,28 @@
+﻿// HyperealVR Defines|SDK_HyperealVR|001
+namespace VRTK
+{
+    using System;
+    using System.Reflection;
+
+    /// <summary>
+    /// Handles all the scripting define symbols for the Hypereal SDK.
+    /// </summary>
+    public static class SDK_HyperealVRDefines
+    {
+        /// <summary>
+        /// The scripting define symbol for the Hypereal SDK.
+        /// </summary>
+        public const string ScriptDefineSymbol = SDK_ScriptingDefineSymbolPredicateAttribute.RemovableSymbolPrefix +
+            "SDK_HYPEREALVR";
+
+        private const string BuildTargetGroupName = "Standalone";
+
+        [SDK_ScriptingDefineSymbolPredicate(ScriptDefineSymbol, BuildTargetGroupName)]
+        [SDK_ScriptingDefineSymbolPredicate(SDK_ScriptingDefineSymbolPredicateAttribute.RemovableSymbolPrefix +
+            "HYPEREALVR_PLUGIN_1_0_OR_NEWER", BuildTargetGroupName)]
+        private static bool IsPluginVersion10OrNewer()
```

Comment:
No need for versioned symbol, so rename to `IsHyperealAvailable` instead. [See the Ximmerse defines file for an example.](https://github.com/thestonefox/VRTK/blob/master/Assets/VRTK/SDK/Ximmerse/SDK_XimmerseDefines.cs#L16)